pom.xml

```diff
@@ -1223,12 +1223,6 @@
                 <artifactId>bcprov-jdk18on</artifactId>
                 <version>${bouncycastle.version}</version>
             </dependency>
-            <!-- convert relaxng to xsd -->
-            <dependency>
-                <groupId>org.relaxng</groupId>
-                <artifactId>trang</artifactId>
-                <version>${trang.version}</version>
-            </dependency>
```

Comment:
C'est inutile ?

Reply:
C'est juste qu'on a la dépendance deux fois :

```
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for fr.gouv.vitamui:commons-parent:pom:8.0.0-SNAPSHOT
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.relaxng:trang:jar -> version ${thaiopensource.trang.version} vs ${trang.version} @ fr.gouv.vitamui:parent:8.0.0-SNAPSHOT, /var/lib/jenkins/workspace/build_vitamui-build_PR-1920/pom.xml, line 1227, column 25
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for fr.gouv.vitamui:parent:pom:8.0.0-SNAPSHOT
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.relaxng:trang:jar -> version ${thaiopensource.trang.version} vs ${trang.version} @ line 1227, column 25
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
```
J'en ai profité pour mettre la version la plus récente des deux.